spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
Here you are doing some kind of integration tests, what we need here is unit tests, in other words, you should test each line of code in the `users_controller.rb`. So:
1) Test that `@user` has been assign
2) Test that the assigned user is the one you want
3) Test it returns a code 200

Reply:
This comment was not fixed